application/src/main/java/org/opentripplanner/graph_builder/module/osm/VertexGenerator.java

```diff
@@ -95,6 +102,13 @@ IntersectionVertex getVertexForOsmNode(OsmNode node, OsmWithTags way) {
         iv = bv;
       }
 
+      if (includeOsmSubwayEntrances && node.isSubwayEntrance()) {
+        String ref = node.getTag("ref");
+
+        boolean accessible = node.isTag("wheelchair", "yes");
```

Comment:
This has three states instead of two, use the unknown one when the tag isn't specified.